tests/test_db.py

```
@@ -1,23 +1,139 @@
-from unittest.mock import Mock
+from unittest.mock import AsyncMock, MagicMock, Mock, patch
+
+import pytest
+from fastapi import HTTPException
+from fastapi.exceptions import RequestValidationError
 
 from viewport.models import db
 
 
+def _mock_db_session(monkeypatch, *, transaction_active: bool = False):
```

Ruff (0.15.21)

[warning] Missing return type annotation for private function `_mock_db_session`

(ANN202)